pkg/api/types.go

```diff
@@ -1200,6 +1210,11 @@ func (a *AgentPoolProfile) IsCoreOS() bool {
 	return a.OSType == Linux && a.Distro == CoreOS
 }
 
+// IsVHDDistro returns true if the distro uses VHD SKUs
+func (a *AgentPoolProfile) IsVHDDistro() bool {
```

Comment:
Too bad this logic has to be the same as the master version, but I guess there isn't a common interface.

Reply:
Indeed, we have this debt among all of the profile methods across master vs agent pool.